### ISSUE TITLE
zen_files.py

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#185]
```
There's an infinite loop in locate_file(), on line 37:
parent = os.path.dirname(editor_file)
should be
parent = os.path.dirname(parent)

But there still will be an infinite loop because os.path.dirname('/') == '/'.

I patched it here:
http://github.com/fmarcia/zen-coding-gedit/commit/6a473046b80ba63de7705c6a55082b
750f206474#diff-1

hth
```

---

Original issue reported on code.google.com by `franck.marcia@gmail.com` on 26 May 2010 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 24 Jan 2011 at 11:19
- Added labels: **Milestone-0.7**

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 24 Jan 2011 at 11:19
- Changed state: **Fixed**
